panimg/contrib/wsi_dcm_to_tiff/dcm_to_tiff.py

```diff
@@ -228,36 +228,35 @@ def dcm_to_tiff(input_dir, output_path):
 
     with TiffWriter(output_path, bigtiff=True) as tif:
         tile_size = image.levels[0].default_instance.tile_size.width
+        test_tile = np.array(image.read_tile(0, (0, 0)))
 
         def tiler(getter, level, cols, rows):
             for row in range(rows):
                 for col in range(cols):
                     im = np.array(getter(level, (col, row)))
                     yield im
 
-        for level in range(0, len(image.levels.levels)):
-            cols = int(math.ceil(image.levels[level].size.width / tile_size))
-            rows = int(math.ceil(image.levels[level].size.height / tile_size))
+        for level in image.levels:
+            cols = int(math.ceil(level.size.width / tile_size))
+            rows = int(math.ceil(level.size.height / tile_size))
```

Comment:
Could you add/update a test so that we're sure this does not regress? Are you certain that this fixes all issues with DICOM WSI?